responsibilities/community_communications.md

```diff
@@ -0,0 +1,24 @@
+# Community communications
```

Comment:
I don't see any link from the responsibilities README to this document, shouldn't it have a link?

Reply:
Link added